### ISSUE TITLE
[Bug Report][3.0.0-alpha.12] Multiple layout items (e.g. v-app-bar & v-navigation-drawer) breaks with nuxt3 SSR; Error: "Layout item "layout-item-0" is missing from layout prop" 

### ISSUE THREAD

**Issue by jhughes-dev** [vuetifyjs/vuetify#14713]
### Environment
**Vuetify Version:** 3.0.0-alpha.12
**Vue Version:** 3.2.31
**Browsers:** Chrome 98.0.4758.80
**OS:** Mac OS 10.15.7, Windows

### Steps to reproduce
clone the reproduction repo, 
```
npm install
npm run dev
```
Load the page and the error appears in the console.

### Expected Behavior
Both `<v-app-bar />` and `<v-navigation-drawer />` should work together.

### Actual Behavior
Console Error, page will not launch

### Reproduction Link
<a href="https://github.com/jhughes-dev/vuetify3_nuxt_layout_item_bug" target="_blank">https://github.com/jhughes-dev/vuetify3_nuxt_layout_item_bug</a>

### Other comments
Note: Sometimes if you add the second layout item after having first launched the app, both items will appear on the page. However, if you restart the app, the error will show up.

Here is the stack for the error:
```
  at ReactiveEffect.fn (.nuxt/dist/server/server.mjs:8556:30)
  at ReactiveEffect.run (node_modules/@vue/reactivity/dist/reactivity.cjs.js:150:25)
  at ComputedRefImpl.get value [as value] (node_modules/@vue/reactivity/dist/reactivity.cjs.js:1090:39)
  at Proxy.<anonymous> (.nuxt/dist/server/server.mjs:21428:61)
  at renderComponentRoot (node_modules/@vue/runtime-core/dist/runtime-core.cjs.js:887:44)
  at renderComponentSubTree (node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:275:51)
  at renderComponentVNode (node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:214:16)
  at renderVNode (node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:310:22)
  at renderVNodeChildren (node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:325:9)
  at renderVNode (node_modules/@vue/server-renderer/dist/server-renderer.cjs.js:302:13)
```

Edit: I also tried reproducing this with `vite` alone and couldn't

**Comment by korkt-kim** [comment]
beta is coming soon.. so just be patient

**Comment by jhughes-dev** [comment]
No worries. Not complaining, just reporting. =)

**Comment by nekosaur** [comment]
Getting error when trying to open the page

```js
m.default is not a function
at file://./.nuxt/dist/server/server.mjs:10:126
at processTicksAndRejections (internal/process/task_queues.js:95:5)
at async renderToString (file://./node_modules/vue-bundle-renderer/dist/index.mjs:247:19)
at async renderMiddleware (file://./.nuxt/nitro/index.mjs:206:20)
at async handle (file://./node_modules/h3/dist/index.mjs:601:19)
```

Tried both node 14 and 16, npm and yarn

**Comment by jhughes-dev** [comment]
I added the sass dependencies back in. I was seeing an unrelated error for that, but I still saw the "layout-item" error once those were in.  
I am away from my linux machine for the weekend. I'll give it a try tomorrow. I double checked removing .nuxt and node_modules to make sure it wasn't some kind of caching error, but I still see the same thing. Two other people reported seeing the same error in the discord chat, but as far as I could tell, neither of them created an issue/repro. /shrug

Maybe there's another dependency I am not aware of. I can try in safari, etc later too, but since it was erroring from the console I didn't think it would be browser dependent.

**Comment by jhughes-dev** [comment]
Just checked, same error on Safari.

**Comment by nekosaur** [comment]
This is the same issue I'm getting https://github.com/nuxt/framework/issues/2679 but I'm running a nuxt version from after the revert so no idea what's going on 🤷

**Comment by jhughes-dev** [comment]
Having computer issues the last few days so I didn't get any chance to check linux on this, sorry.

**Comment by jhughes-dev** [comment]
Update, I was able to get this symptom to reproduce on linux (Ubuntu 20.04, same node/npm, Firefox browser) I was using nuxt3@latest you might have to install the specific version to the project.

**Comment by martinEtflais** [comment]
I had a similar issue using alpha 12 with laravel 9 and ssr using inertiajs. I managed to make it running with try and fail method changing vappbar and navigationdrawer (using two of them on the same level - left and right) order but could not make it work in my desired layout :-/ nevertheless ssr error turned from this to voverlay trying to attach listeners to the non existing window object... Guess its just too soon to think about working ssr with vuetify 3 at the current state of development... Working quite good client side though. Just keep seeing this wierd bug with floating labels of inputs... Look like some kind of font size / element switching problem... Transition doesnt look smooth and font seems like flashing in matter of size... Never seen this bug on any codepen or in the examples in docs but personally had in every fun / test project I` ve tried to made with alpha 12.

**Comment by johnleider** [comment]
fixed by #14747
